translations/en.json

```diff
@@ -978,7 +978,5 @@
     "mobile-network-go-to-settings": "Go to settings",
     "mobile-network-use-mobile-data": "Status tends to use a lot of data when syncing chats. You can choose not to sync when on mobile network",
     "status-not-sent": "",
-    "status-not-sent-without-tap": "",
-    "confirm-install": "Confirm Install",
-    "extension-install-alert": "Development mode is required to install an extension. Do you want to enable and continue installing?"
+    "status-not-sent-without-tap": ""
```

Comment:
looks like this was removed during rebasing

Reply:
It doesn't seem to exist in the develop branch anymore.

Reply:
i mean last two items "confirm-install" and "extension-install-alert" we have them in develop https://github.com/status-im/status-react/blob/8e1da91ea01b7eb0d7fd5203df4485a12f94f7bf/translations/en.json#L982